### PR TITLE
Make use of postgres package

### PR DESCRIPTION
**Pull request by bradsawadye**
Update the Jempi service to use the postgres database package

## Summary by CodeRabbit

- **New Features**
  - Introduced PostgreSQL database initialization scripts that create tables and insert initial data.
  - Added new PostgreSQL configuration services for `client-registry-jempi` and `dashboard-visualiser-superset`.

- **Configuration Updates**
  - Updated environment variables for PostgreSQL configurations across multiple Docker Compose files.
  - Added new PostgreSQL service definitions and network configurations.
  - Removed outdated PostgreSQL service configurations for better performance and maintenance.

- **Dependency Updates**
  - Added PostgreSQL-related dependencies in `package.json` files for database configuration.

- **Script Enhancements**
  - Enhanced `swarm.sh` scripts to deploy PostgreSQL config importers for database setup.

### CODE REVIEW

**Review by brett-onions — APPROVED** [review]
Look Good to Me

File: client-registry-jempi/importer/postgres/create-db.js [hunk 1]
```diff
@@ -0,0 +1,136 @@
+const { Pool } = require('pg');
+
+const user = process.env.POSTGRES_USER || 'postgres'
+const host = process.env.POSTGRES_SERVICE || 'postgres-1'
+const database = process.env.POSTGRES_DATABASE || 'jempi'
+const password = process.env.POSTGRES_PASSWORD || 'instant101'
+const port = process.env.POSTGRES_PORT || 5432
+const newDb = process.env.NEW_DATABASE_NAME || 'jempi'
+const newUser = process.env.NEW_DATABASE_USER || 'keycloak'
```
Comment (drizzentic) [inline client-registry-jempi/importer/postgres/create-db.js:9]: shouldn't this be jempi user to match the .env.cluster configs

**Review by drizzentic — CHANGES_REQUESTED** [review]
LGTM! We just need to ensure tests are also refactored to work with new package. Also left a small minor comment

File: client-registry-jempi/importer/postgres/create-db.js (hunk 1 above)
Comment (bradsawadye) [reply client-registry-jempi/importer/postgres/create-db.js:9]: It maps to that. check the docker compose file https://github.com/jembi/platform/pull/308/files#diff-3452b9e7934f8ae11c4a9771e08c298a7a0f1eea4c703f0c770e39a374e66067R20-R22

Comment (drizzentic) [reply client-registry-jempi/importer/postgres/create-db.js:9]: I actually mean the default value
```suggestion
const newUser = process.env.NEW_DATABASE_USER || 'jempi'
```

**Review by drizzentic — APPROVED** [review]

**Pull request merged**